tests/ui/specs/DayPicker/DayPicker-GetDayString-specs.js

```diff
@@ -0,0 +1,175 @@
+const Page = require('./DayPickerPage');
+
+describe('DayPicker', function () {
+	beforeEach(async function () {
+		await Page.open('GetDayString');
+	});
+
+	describe('LTR', function () {
+		const dayPicker = Page.components.getDayStringDayPicker;
+
+		it('should selected day display [QWTC-2427]', async function () {
```

Comment:
Would you please update TC description? 
```suggestion
		it('should display proper selected day string [QWTC-2427]', async function () {
```

Reply:
Suggested change.